graylog2-web-interface/src/components/graylog/ListGroupItem.md

```diff
@@ -1,59 +1,59 @@
 ### Normal List
 ```js
 import { ListGroup } from 'components/graylog';
-const styles = ['Default', 'Danger', 'Warning', 'Success', 'Info'];
 
 <ListGroup>
-  {styles.map((style) => (
-    <ListGroupItem key={`lgi-${style}`} bsStyle={style.toLowerCase()}>{style}</ListGroupItem>
-  ))}
+  <ListGroupItem>Item 1</ListGroupItem>
```

Comment:
Updated examples to use same as [react-bootstrap v3](https://react-bootstrap-v3.netlify.com/components/list-group/)